wai-language-server/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "wai-language-server"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde_json = "1.0.91"
+tokio = { version = "1.25.0", features = ["full"] }
```

I think we should be able to trim the features we use and save a bit on compilation times.

```suggestion
tokio = { version = "1.25.0", features = ["rt", "rt-multi-thread", "macros"] }
```

---

We also needed the "io-std" feature